libtest/cr_template.h

```diff
@@ -22,13 +22,16 @@
  *
  */
 
-#ifndef LIBTEST_TEMPLATE_LIB_H_INCLUDED
-#define LIBTEST_TEMPLATE_LIB_H_INCLUDED 1
+#ifndef LIBTEST_CR_TEMPLATE_H_INCLUDED
+#define LIBTEST_CR_TEMPLATE_H_INCLUDED 1
 
-#include "testutils.h"
+#include "syslog-ng.h"
```

Comment:
Why do we need to include syslog-ng.h?

Reply:
I think to show dependencies explicitly: syslog-ng.h contains definition of LogMessage.
Of course other headers, e.g. templates.h includes it.